README.rst

```diff
@@ -327,28 +351,38 @@ For example, the following will build Kokkos for NVIDIA A100 cards
         -DKokkos_ENABLE_TESTS:BOOL=OFF \
         -DKokkos_ENABLE_LIBDL:BOOL=OFF
     cmake --build build && cmake --install build
-    echo export CMAKE_PREFIX_PATH=/opt/kokkos/4.1.00/AMPERE80:\$CMAKE_PREFIX_PATH
+    export CMAKE_PREFIX_PATH=/opt/kokkos/4.x.y.z/AMPERE80:$CMAKE_PREFIX_PATH
+
 
 Next, append the install location to ``CMAKE_PREFIX_PATH``.
 Note that the C++20 standard is required (``-DCMAKE_CXX_STANDARD=20`` option), and hence CUDA v12 is required for the CUDA backend.
-If an installation is not found, our builder will clone and install it during the build process.
 
-The simplest way to install Lightning-Kokkos (OpenMP backend) through ``pip``.
+Install Lightning-Kokkos
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+If an installation of Kokkos is not found, then our builder will clone and install it during the build process. Lightning-Qubit should be installed (compilation is not necessary):
 
-.. code-block:: console
+The simplest way to install Lightning-Kokkos (OpenMP backend) through ``pip``.
 
-   PL_BACKEND="lightning_kokkos" python scripts/configure_pyproject_toml.py
-   CMAKE_ARGS="-DKokkos_ENABLE_OPENMP=ON" python -m pip install .
+.. code-block:: bash
 
-To build the plugin directly with CMake as above:
+    git clone https://github.com/PennyLaneAI/pennylane-lightning.git
+    cd pennylane-lightning
+    PL_BACKEND="lightning_qubit" python scripts/configure_pyproject_toml.py
+    SKIP_COMPILATION=True pip install -e .
+    PL_BACKEND="lightning_kokkos" python scripts/configure_pyproject_toml.py
+    CMAKE_ARGS="-DKokkos_ENABLE_OPENMP=ON" python -m pip install -e . -vv
 
-.. code-block:: console
+The supported backend options are 
 
-   cmake -B build -DKokkos_ENABLE_OPENMP=ON -DPL_BACKEND=lightning_kokkos -G Ninja
-   cmake --build build
+- ``SERIAL``
+- ``OPENMP``
+- ``THREADS``
+- ``HIP``
+- ``CUDA`` 
```

Comment:
If you are not going to provide some details for each of these backend kernels, I still think that reviewing them in one line is better than:
![image](https://github.com/user-attachments/assets/c7463e8e-a2be-42e5-97ff-2f9b127df866)

Reply:
An alternative 
![Screenshot from 2024-08-15 12-06-48](https://github.com/user-attachments/assets/ddb27219-c15e-4503-8a65-406b8f6de66b)
 :smile: